### ISSUE TITLE
Using console.log in `email/index.ts` does not log anything

### ISSUE THREAD

**Issue by cymen** [successor-software/mailing#90]
While digging into https://github.com/successor-software/mailing/issues/89 it was confusing to not be able to log anything from `email/index.ts`.

**Comment by psugihara** [comment]
Hm, I'm seeing it work on my end. Perhaps the file wasn't imported yet? Here's what I see when the index file with `console.log`  gets required by mailing to send an email...

<img width="1440" alt="Screen Shot 2022-08-05 at 8 36 47 AM" src="https://user-images.githubusercontent.com/282016/183111919-3fa26537-5b89-47f2-9eed-26727764f1c0.png">
<img width="493" alt="Screen Shot 2022-08-05 at 8 38 00 AM" src="https://user-images.githubusercontent.com/282016/183112001-3c305eee-5af3-47d9-b49c-e719ac3fdec8.png">

**Comment by cymen** [comment]
I discovered that this is due to having `"isolatedModules": true` in my root `tsconfig.json` under `compilerOptions`. If I comment that out, the console logging works. I discovered this by running:

```sh
yarn ts-node -e 'console.log("test")'
```

And if `isolatedModules` is enabled, there is an error message:

```sh
[eval].ts:1:1 - error TS1208: '[eval].ts' cannot be compiled under '--isolatedModules' because it is considered a global script file. Add an import, export, or an empty 'export {}' statement to make it a module.

1 console.log("test")
  ~~~~~~~

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I don't get any error when running `npx mailing` but at least it makes sense! I only wanted this while figuring out https://github.com/successor-software/mailing/issues/89 so I'll close this as it seems out of scope of mailing although good to know.

**Comment by psugihara** [comment]
got it, thanks for digging in. fyi i'm adding dotenv to the mailing cli right now and will have a new release out later today that will hopefully fix the prob you're having in #89.
